storage-iops-latency-throughput-demo/azuredeploy.json

```diff
@@ -0,0 +1,306 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountType": {
```

Comment:
I think you can move these to a variable and simplify the deployment a bit - will still get at the main purpose of your sample.